fe/fe-core/src/main/java/com/starrocks/qe/ConnectProcessor.java

```diff
@@ -424,6 +425,7 @@ private void dispatch() throws IOException {
         ctx.setCommand(command);
         ctx.setStartTime();
         ctx.setWorkGroup(null);
+        ctx.setErrorCodeOnce("");
```

Comment:
If once, would subsequent `setErrorCodeOnce` work?

Reply:
yes error code can be reset to empty